integration_tests/src/main/python/parquet_write_test.py

```diff
@@ -29,15 +29,21 @@
 coalesce_parquet_file_reader_conf={'spark.rapids.sql.format.parquet.reader.type': 'COALESCING'}
 reader_opt_confs = [original_parquet_file_reader_conf, multithreaded_parquet_file_reader_conf,
         coalesce_parquet_file_reader_conf]
-
+parquet_decimal_gens=[decimal_gen_default, decimal_gen_scale_precision, decimal_gen_same_scale_precision, decimal_gen_64bit]
+parquet_decimal_struct_gen= StructGen([['child'+str(ind), sub_gen] for ind, sub_gen in enumerate(parquet_decimal_gens)])
 writer_confs={'spark.sql.legacy.parquet.datetimeRebaseModeInWrite': 'CORRECTED',
               'spark.sql.legacy.parquet.int96RebaseModeInWrite': 'CORRECTED'}
 
+
+parquet_basic_gen =[byte_gen, short_gen, int_gen, long_gen, float_gen, double_gen,
+                    string_gen, boolean_gen, date_gen,
+                    # we are limiting TimestampGen to avoid overflowing the INT96 value
```

Comment:
Spark doesn't overflow and writes the value as a binary. 

Thanks for pointing this out. I have made the change to throw an exception in case of overflow. PTAL

Reply:
@razajafri we need a follow on issue to "fix" this eventually. It might not ever be a high enough priority to actually fix, but we should still have it on the backlog.

@jlowe what do you think? In a query we are never really writing more than one dataset per query so if they hit the error then can turn off parquet writes. Part of me wants a separate config, but another part says that is complexity for no real benefit.